### PR TITLE
Refactor activity report review pages

### PR DESCRIPTION
**Pull request by jasalisbury**
**Description of change**
 * Each page has more control over what shows up on it's review page
 * Topics and resources shows download links to the files that have been uploaded to the report

**How to test**

1) Pull down, fill out a report
2) Add files to the report
3) Look at the files on the review page. You may have to invert the logic in `frontend/src/pages/ActivityReport/Pages/Review/FileReviewItem.js` line 16 in order to see an actual download link (unless https://github.com/adhocteam/Head-Start-TTADP/pull/176 is merged). We only show the download link for approved files and files are never currently approved.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/247

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

**Review by dcmcand — APPROVED** [review]
Looks good. This should make customizing the review page much easier!

**Review by rahearn — CHANGES_REQUESTED** [review]
There's an issue where if you upload a file, click save and continue, and then jump right to the review page you trigger a pretty fatal error:

Screen Shot 2021-03-01 at 3 28 12 PM

**Comment by jasalisbury** [comment]
@dcmcand requested a re-review because I made a few file upload changes in my latest commit you might want to checkout.

**Review by dcmcand — COMMENTED** [review]
The file stuff looks good. I like returning a url right away. There is something funky with printing though. It looks like only the first file shows up when you print.

**Review by dcmcand — CHANGES_REQUESTED** [review]
There is something funky with printing though. It looks like only the first file shows up when you print.

**Comment by jasalisbury** [comment]
> There is something funky with printing though. It looks like only the first file shows up when you print.

Can you be more specific on how you were able to only see a single file? I was able to see two files here:
Screenshot from 2021-03-02 17-31-03

**Comment by dcmcand** [comment]
I also tried it  with a completed report. As you can see, I had four files attached.
Screenshot from 2021-03-03 09-16-27
If you look at the attached report, only one resource and no attachments show up.
[Review and submit - Activity Report - TTA Smart Hub.pdf](https://github.com/adhocteam/Head-Start-TTADP/files/6076536/Review.and.submit.-.Activity.Report.-.TTA.Smart.Hub.pdf)

**Comment by jasalisbury** [comment]
@dcmcand Print should be working correctly now

**Review by dcmcand — APPROVED** [review]
Nice!

File: docker-compose.yml [hunk 1]
```diff
@@ -27,7 +27,9 @@ services:
   aws-cli:
     image: amazon/aws-cli
     env_file: .env
-    command: ["--endpoint-url", "$S3_ENDPOINT", "s3api", "create-bucket", "--bucket", "$S3_BUCKET"]
+    command: ["--endpoint-url", "http://minio:9000", "s3api", "create-bucket", "--bucket", "$S3_BUCKET"]
```
Comment (rahearn) [inline docker-compose.yml:30]: Sorry for tagging along on your PR, @jasalisbury. I found this while reviewing this PR, but the change it totally unrelated to your changes.

**Review by rahearn — APPROVED** [review]
🎉

**Pull request merged**